Editor/EditorCore/PolyShapeEditor.cs

```diff
@@ -750,10 +750,10 @@ void OnFinishVertexMovement()
 
         void UndoRedoPerformed()
         {
-            if (polygon.polyEditMode == PolyShape.PolyEditMode.None)
-                ProBuilderEditor.selectMode = ProBuilderEditor.selectMode & ~(SelectMode.InputTool);
-            else
-                ProBuilderEditor.selectMode = ProBuilderEditor.selectMode | SelectMode.InputTool;
+            // if (polygon.polyEditMode == PolyShape.PolyEditMode.None)
+            //     ProBuilderEditor.selectMode = ProBuilderEditor.selectMode & ~(SelectMode.InputTool);
+            // else
+            //     ProBuilderEditor.selectMode = ProBuilderEditor.selectMode | SelectMode.InputTool;
```

Comment:
what's going on here?

Reply:
> on macos the cursor size is incorrect
> 
> also to note - the cursor images are 512^2, which is way larger than they need to be in the first place. importing icons is the main cause for lengthy package import times.

The cursor images are now 64x64, feel free to modify the aspect on MacOS, I am on Windows and don't really know how to handle that in Mac :p

Reply:
This is not valid anymore after the merge with the new polyshape as editortool